apis/Google.Cloud.Logging.NLog/Google.Cloud.Logging.NLog.Tests/GoogleStackdriverTargetTest.cs

```diff
@@ -171,16 +171,43 @@ private LogEventInfo CreateLog(LogLevel level, string msg)
                 {
                     googleTarget.SendJsonPayload = true;
                     googleTarget.ContextProperties.Add(new TargetPropertyWithContext() { Name = "Galaxy", Layout = "Milky way" });
-                    LogManager.GetLogger("testlogger").Info("Hello {Planet}", "Earth");
+                    LogManager.GetLogger("testlogger").Info("Hello {Planet}, width {Radius} km", "Earth", 6371);
                     return Task.FromResult(0);
                 }, includeEventProperties: true);
             Assert.Single(uploadedEntries);
             var entry0 = uploadedEntries[0];
-            Assert.Equal(string.Empty, entry0.TextPayload?.Trim() ?? string.Empty);
-            Assert.Equal("Hello \"Earth\"", entry0.JsonPayload.Fields["message"].StringValue);
-            Assert.Equal(2, entry0.JsonPayload.Fields["properties"].StructValue.Fields.Count);
+            Assert.Equal("", entry0.TextPayload?.Trim() ?? "");
+            Assert.Equal("Hello \"Earth\", width 6371 km", entry0.JsonPayload.Fields["message"].StringValue);
+            Assert.Equal(3, entry0.JsonPayload.Fields["properties"].StructValue.Fields.Count);
             Assert.Equal("Earth", entry0.JsonPayload.Fields["properties"].StructValue.Fields["Planet"].StringValue);
             Assert.Equal("Milky way", entry0.JsonPayload.Fields["properties"].StructValue.Fields["Galaxy"].StringValue);
+            Assert.Equal(6371, entry0.JsonPayload.Fields["properties"].StructValue.Fields["Radius"].NumberValue);
+        }
+
+        [Fact]
+        public async Task SingleLogEntryWithJsonCollectionProperties()

```

Comment:
Nice extra test. Please also test `boolean` for completeness. I think this'll mean that all supported types are in this test.

Reply:
Fixed